webpack/es-bundle-core.babel.js

```diff
@@ -29,9 +29,10 @@ const result = configBuilder(
       ],
     },
     output: {
-      globalObject: "this",
-      library: "SwaggerUIBundle",
+      // with webpack 5 and commonjs2, there should not be a library[String] or library[Object]
+      // https://github.com/webpack/webpack/issues/12675
       libraryTarget: "commonjs2",
+      libraryExport: "default"
```

Comment:
I did try to use following configuration and it produced a valid bundle which I verified by test using following code:

**Configuration**:

```suggestion
      library: {
        type: "commonjs2",
        export: "default",
      },
```

**Test**:

```js
const SwaggerUI = require('./dist/swagger-ui-es-bundle-core.js');
  
console.dir(SwaggerUI);
```

Reply:
ok. appears to be working in CRA demo for me.